src/bundle/Resources/public/js/scripts/admin.search.autocomplete.content.js

```diff
@@ -0,0 +1,40 @@
+(function (global, doc, ibexa, Routing) {
+    const globalSearch = doc.querySelector('.ibexa-global-search');
+
+    if (!globalSearch) {
+        return;
+    }
+
+    const { escapeHTML } = ibexa.helpers.text;
+    const { highlightText } = ibexa.helpers.highlight;
+    const contentTypeHelper = ibexa.helpers.contentType;
```

Comment:
```suggestion
    const { getContentTypeIconUrl, getContentTypeName } = ibexa.helpers.contentType;
```

and line 31,32 just `getContentTypeIconUrl()` and `getContentTypeName()`